site/en/docs/extensions/mv2/getstarted/index.md

```diff
@@ -328,6 +328,11 @@ The extension is now fully functional! Reload the extension, refresh this page,
 click the button to turn it green! However, some users may want to change the background to a
 different color.
 
+{% Aside 'gotchas' %}
+You can't run a content script on an internal Chrome page, such as "chrome://extensions". Be sure
+to try out the extension on a real webpage, like https://google.com.
```

Comment:
```suggestion
Extensions can not inject content scripts on internal Chrome pages like "chrome://extensions". Be
sure to try out the extension on a real webpage like [https://google.com](https://google.com).
```

Rephrase slightly to emphasise that this is a platform restriction, not an issue with the reader.